tools/topology/topology2/sof-ace-tplg/CMakeLists.txt

```diff
@@ -22,6 +22,11 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-rt711-4ch.bin,DEEPBUFFER_FW_DMA_MS
 
 "cavs-rt5682\;sof-mtl-max98357a-rt5682\;PLATFORM=mtl,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-max98357a-rt5682.bin,DEEPBUFFER_FW_DMA_MS=10"
+
+"cavs-rt5682\;sof-mtl-max98357a-rt5682-ssp2-ssp0\;PLATFORM=mtl,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,\
+PDM1_MIC_B_ENABLE=1,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-mtl-max98357a-rt5682.bin,\
+DEEPBUFFER_FW_DMA_MS=10,HEADSET_SSP_DAI_INDEX=2,SPEAKER_SSP_DAI_INDEX=0,HEADSET_CODEC_NAME=SSP2-Codec,\
+SPEAKER_CODEC_NAME=SSP0-Codec"
```

Comment:
the only nit-pick I would have is that the information is duplicated with HEADSET_SSP_DAI_INDEX=2 and HEADSET_CODEC_NAME=SSP2-Codec

If you need to change that's 2 variables instead of a single value.

This can be fixed as an improvement, it's good enough for now.

Reply:
@plbossart that feature is coming. @jsarha is working on a string concat feature that will allow us to do this.